configuration/messageproperties/messages_zl_en.properties

```diff
@@ -46,3 +46,5 @@ ui.i18n.Role.name.c1964e06-f672-4e0e-864c-f5124ae67dba=HIV Pharmacist
 ui.i18n.Role.name.6a45d1bd-0445-466e-b73e-3d8de0c8e076=HIV Lab Technician
 ui.i18n.Role.name.d07509d0-2b5a-48fc-b298-b3f00830d44e=HIV Social Worker
 ui.i18n.Role.name.ec236c43-c086-4139-8ace-5d7ba5472d2f=HIV NEC
+
+pih.app.labs.label=Lab Order Module
```

Comment:
Module?  Is this what we want this to read?

Reply:
Yes... it doesn't really make a ton of sense to me, but that was the request from the team on the ground.  It's only being applied to Haiti so I'm fine with running with it for now.  Again, hopefully we will get this all consolidated and cleaned up at some point.